doc/examples/transform/plot_rescale.py

```diff
@@ -27,8 +27,8 @@
 
 image = color.rgb2gray(data.astronaut())
 
-image_rescaled = rescale(image, 1.0 / 4.0, anti_aliasing=False)
-image_resized = resize(image, (image.shape[0] / 4, image.shape[1] / 4),
+image_rescaled = rescale(image, 1.0 // 4.0, anti_aliasing=False)
```

Comment:
I'm ok with the line below, but this one is surely incorrect?

```python
In [1]: 1.0 // 4.0
Out[1]: 0.0
```

Reply:
You're right; I fell into an error in resize; and didn't quite realize that we also had rescale.  My mistake!

Reply:
```suggestion
image_rescaled = rescale(image, 0.25, anti_aliasing=False)
```